### ISSUE TITLE
Scoreboard & placeholders. leaderboards - extra ideas

### ISSUE THREAD

**Issue by funtodead** [kangarko/CoreArena#276]
hello, so I'm assuming you guys did get some questions about this.
but I just wanted to say what I think should be added to make it better.
So firstly for placeholders

○ Mob-kills
○ Nuggets
○ current phase level (ig)
○ what game is about to start/time 
○ level (ig)
○ class (ig)

Now leaderboards
○ top phases 
○ top mob kills
○ top nuggets
○ top clutches (if everyone died and you're alone)

and scoreboard

○ nuggets
○ next game / time left
○ class
○ maps ig/not ig

And now. 2 ideas I made by myself using plugins
but could be a great addon

1. PLAY GUI
![CORE1](https://user-images.githubusercontent.com/58358289/74587378-6d285700-4ffa-11ea-9a0d-5003b4b7bd22.png)

now there are two choices of how to make that.
I personally codded it, but im sure not everyone can code so it could be good
basically all it does is
 just runs the command
/ma join (arena)

but how i think it should add it to the /ma menu
and make it so you can change item,name,lore,and then pick arena from arenas list 

and now last is
ITEMS, i used itemJoin plugin to do this
but this could be very smart to add
look in pictures:
this opens the gui i codded
![core2](https://user-images.githubusercontent.com/58358289/74587415-c1cbd200-4ffa-11ea-96e3-45a81debd8d8.png)

this opens the /Ma rewards and you can use it as a shop
![core3](https://user-images.githubusercontent.com/58358289/74587431-db6d1980-4ffa-11ea-8afa-4eb066a3a605.png)

Thank you for reading this!,
i hope you take this into consideration 
funtodead.

**Comment by JavaFactoryDev** [comment]
Hi funtodead!
Thanks so much for suggesting your feature/idea. We have carefully discussed it with our team and have concluded that, although interesting, it is at present, not a good fit nor we have the resources to add it in the near future. Feel free to re-suggest this feature in the future if you still have this problem

Regards
Leonhard from MineAcademy
